rotkehlchen/tests/exchanges/test_kucoin.py

```diff
@@ -763,3 +764,24 @@ def mock_api_query_response(case, options):  # pylint: disable=unused-argument
         )
 
     assert asset_movements == expected_asset_movements
+
+
+@pytest.mark.parametrize('should_mock_current_price_queries', [True])
+def test_query_old_trades_sandbox(sandbox_kuckoin, inquirer):  # pylint: disable=unused-argument
```

Comment:
```suggestion
def test_query_old_trades_sandbox(sandbox_kucoin, inquirer):  # pylint: disable=unused-argument
```

Fix here and elsewhere

Reply:
lol, is not mine but I didn't notice